README.md

```diff
@@ -4,9 +4,13 @@ OSPSuite.ReportingEngine implementation in R
 
 <!-- badges: start -->
 
+  ![](https://img.shields.io/github/downloads/Open-Systems-Pharmacology/Suite/latest/total?label=Downloads%20latest%20release&logo=data:image/svg%2bxml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0idXRmLTgiPz4KPCEtLSBMaWNlbnNlOiBQRC4gTWFkZSBieSBNYXJ5IEFrdmVvOiBodHRwczovL21hcnlha3Zlby5jb20vIC0tPgo8c3ZnIGZpbGw9IiMwMDAwMDAiIHdpZHRoPSI4MDBweCIgaGVpZ2h0PSI4MDBweCIgdmlld0JveD0iMCAwIDI0IDI0IiBpZD0iZG93bmxvYWQtNSIgZGF0YS1uYW1lPSJMaW5lIENvbG9yIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGNsYXNzPSJpY29uIGxpbmUtY29sb3IiPjxwb2x5bGluZSBpZD0ic2Vjb25kYXJ5IiBwb2ludHM9IjE1IDE0IDEyIDE3IDkgMTQiIHN0eWxlPSJmaWxsOiBub25lOyBzdHJva2U6IHJnYigyNTUsIDI1NSwgMjU1KTsgc3Ryb2tlLWxpbmVjYXA6IHJvdW5kOyBzdHJva2UtbGluZWpvaW46IHJvdW5kOyBzdHJva2Utd2lkdGg6IDI7Ij48L3BvbHlsaW5lPjxsaW5lIGlkPSJzZWNvbmRhcnktMiIgZGF0YS1uYW1lPSJzZWNvbmRhcnkiIHgxPSIxMiIgeTE9IjE3IiB4Mj0iMTIiIHkyPSIzIiBzdHlsZT0iZmlsbDogbm9uZTsgc3Ryb2tlOiByZ2IoMjU1LCAyNTUsIDI1NSk7IHN0cm9rZS1saW5lY2FwOiByb3VuZDsgc3Ryb2tlLWxpbmVqb2luOiByb3VuZDsgc3Ryb2tlLXdpZHRoOiAyOyI+PC9saW5lPjxwYXRoIGlkPSJwcmltYXJ5IiBkPSJNNCwxN3YzYTEsMSwwLDAsMCwxLDFIMTlhMSwxLDAsMCwwLDEtMVYxNyIgc3R5bGU9ImZpbGw6IG5vbmU7IHN0cm9rZTogcmdiKDI1NSwgMjU1LCAyNTUpOyBzdHJva2UtbGluZWNhcDogcm91bmQ7IHN0cm9rZS1saW5lam9pbjogcm91bmQ7IHN0cm9rZS13aWR0aDogMjsiPjwvcGF0aD48L3N2Zz4=)
+  ![](https://img.shields.io/github/downloads/Open-Systems-Pharmacology/Suite/total?label=Downloads%20total&logo=data:image/svg%2bxml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0idXRmLTgiPz4KPCEtLSBMaWNlbnNlOiBQRC4gTWFkZSBieSBNYXJ5IEFrdmVvOiBodHRwczovL21hcnlha3Zlby5jb20vIC0tPgo8c3ZnIGZpbGw9IiMwMDAwMDAiIHdpZHRoPSI4MDBweCIgaGVpZ2h0PSI4MDBweCIgdmlld0JveD0iMCAwIDI0IDI0IiBpZD0iZG93bmxvYWQtNSIgZGF0YS1uYW1lPSJMaW5lIENvbG9yIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGNsYXNzPSJpY29uIGxpbmUtY29sb3IiPjxwb2x5bGluZSBpZD0ic2Vjb25kYXJ5IiBwb2ludHM9IjE1IDE0IDEyIDE3IDkgMTQiIHN0eWxlPSJmaWxsOiBub25lOyBzdHJva2U6IHJnYigyNTUsIDI1NSwgMjU1KTsgc3Ryb2tlLWxpbmVjYXA6IHJvdW5kOyBzdHJva2UtbGluZWpvaW46IHJvdW5kOyBzdHJva2Utd2lkdGg6IDI7Ij48L3BvbHlsaW5lPjxsaW5lIGlkPSJzZWNvbmRhcnktMiIgZGF0YS1uYW1lPSJzZWNvbmRhcnkiIHgxPSIxMiIgeTE9IjE3IiB4Mj0iMTIiIHkyPSIzIiBzdHlsZT0iZmlsbDogbm9uZTsgc3Ryb2tlOiByZ2IoMjU1LCAyNTUsIDI1NSk7IHN0cm9rZS1saW5lY2FwOiByb3VuZDsgc3Ryb2tlLWxpbmVqb2luOiByb3VuZDsgc3Ryb2tlLXdpZHRoOiAyOyI+PC9saW5lPjxwYXRoIGlkPSJwcmltYXJ5IiBkPSJNNCwxN3YzYTEsMSwwLDAsMCwxLDFIMTlhMSwxLDAsMCwwLDEtMVYxNyIgc3R5bGU9ImZpbGw6IG5vbmU7IHN0cm9rZTogcmdiKDI1NSwgMjU1LCAyNTUpOyBzdHJva2UtbGluZWNhcDogcm91bmQ7IHN0cm9rZS1saW5lam9pbjogcm91bmQ7IHN0cm9rZS13aWR0aDogMjsiPjwvcGF0aD48L3N2Zz4=)

```

Comment:
those are downloads for the Suite, not for RE
I added them just an example, not for the RE-Repo :)